src/components/TopicLoader/TopicLoader.js

```diff
@@ -120,7 +123,9 @@ class TopicLoader extends Component {
 
     if (
       activeTopic &&
-      (language !== prevProps.language || apiKey !== prevProps.apiKey)
+      (language !== prevProps.language ||
+        apiKey !== prevProps.apiKey ||
+        apiKeyName !== prevProps.apiKeyName)
```

Comment:
Can the key and its key name really change?

Reply:
Well I think so if we wait on a fetchApiKey call, no? It changes from undefined to something